docker/neutron/neutron-mlnx-agent/Dockerfile.j2

```diff
@@ -49,4 +50,4 @@ RUN {{ macros.install_pip(neutron_mlnx_agent_pip_packages | customizable("pip_pa
 {% block neutron_mlnx_agent_footer %}{% endblock %}
 {% block footer %}{% endblock %}
 
-USER neutron
+USER root
```

Comment:
Is there a bug that this fixes? Does it need to go upstream?

Reply:
Probably.... at present, neutron user cannot read libvirt socket. Ideally, we would configure eswitchd agent to use tcp socket rather than system socket as nova does. In the meantime, the workaround is to run mlnx-agent container as root.

Reply:
Ok we dont need to run as `root`. User `neutron` is fine as long as libvirt exposes its readonly sockets.